### ISSUE TITLE
Do we need get functions?

### ISSUE THREAD

**Issue by austin-cari** [DiamondBrook/Dungeons-and-Dragons-character-generator#1]
Whenever you call a print statement, you can use the function call charObjName.var to access field paramaters. For instance, if I wanted the name of a character object, I could just say bob.name instead of bob.getName(). So, do we even need to have any get functions in this program?

**Comment by Stargon** [comment]
Yes. An object should have it's field on private so that other objects can't reference or change it's fields.
